src/routes/~(dashboard)/~$team/~$study/~configuration/components/BasicInfoPreview.tsx

```diff
@@ -0,0 +1,71 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Study Platform open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { cn } from "@/utils/cn";
+import type { BasicInfoForm } from "../lib/useBasicInfoForm";
+
+interface BasicInfoPreviewProps {
+  form: BasicInfoForm;
+  highlightedField?: string;
+}
+
+const HighlightOverlay = () => (
+  <div
+    data-testid="phone-preview-highlight"
+    className={cn(
+      "[--offset:10%]",
+      "absolute -top-(--offset) -left-(--offset)",
+      "bg-fill-info/5 border-border-info pointer-events-none size-[calc(100%+var(--offset)*2)] rounded-[calc(var(--phone-width)*0.03)] border-[1.5px]",
+    )}
+  />
+);
+
+export const BasicInfoPreview = ({
+  form,
+  highlightedField,
+}: BasicInfoPreviewProps) => {
+  const title = form.watch("title");
+  const explanation = form.watch("explanation");
```

Comment:
I don't mind keeping as is, just letting you know this notation also exists!

```suggestion
  const [title, explanation] = form.watch(["title", "explanation"]);
```